src/services/widgets.style.service.ts

```diff
@@ -10,6 +10,7 @@ export enum HeaderVariant {
 export enum FooterVariant {
   None,
   Logo,
+  LogoNoYearsRange,
```

Comment:
I know its a little going deep into details but I preferred to change the logo to `LogoWithRange` and the new one to `Logo` instead of `LogoNoYearsRange`